compiler/src/compile.re

```diff
@@ -149,6 +149,7 @@ let next_state = (~is_root_file=false, {cstate_desc, cstate_filename} as cs) =>
       Grain_utils.Config.apply_attribute_flags(
         ~no_pervasives=has_attr("noPervasives"),
         ~runtime_mode=has_attr("runtimeMode"),
+        ~no_exception_mod=has_attr("exceptionMod"),
```

Comment:
What about "noExceptionMod" to keep with the pattern of "noPervasives"? I assume you did this because currently this attribute would only be used on the exception module so it is taken to mean "this is the exception module", but "noExceptionMod" more accurately reflects what the flag is actually doing I feel.

Reply:
I had thought of `noExceptions` but I thought that was too ambiguous, but I've come around on it.

Reply:
Updated.